### PR TITLE
created example api/routes [MB-3] [MB-4] [MB-5]

### PR DESCRIPTION
**Pull request by khreate**
* **Please check if the PR fulfills these requirements**
- [✅] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
This PR creates three new features in a new API that interfaces with the people example API.
These features are routes to get all of the people in the API, remove a person from the API, and add a person to the API.

* **What is the current behavior?What does the code do before you made the new changes?** 
Previous to these changes, the code did nothing (index.js empty)

* **What is the new behavior?**
This code when executed will run an API.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
This PR requires the user to run the api start command from their terminal.

### CODE REVIEW

File: SK/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    .catch(function(error){ // on error, the API response is "USER NOT DELETED"
+        console.log(error)
+        res.json({"message": "ERROR: USER NOT DELETED"})
+    }) 
+});
+
+app.post('/addPerson', function(req, res){ // route which adds a person to the API 
+    const body = req.body;  
+    console.log(body) // logs the body of the post request for debugging
+    axios.post("http://java-sample-api-2020.herokuapp.com/addPerson", bodyPost) // POST request attempted here
```
Comment (Codubee) [inline SK/index.js]: The post did not work here. Might have to try this one out and see if it works.

**Review by Codubee — CHANGES_REQUESTED** [review]
Left a comment

**Review by khreate — COMMENTED** [review]
Fixed a variable name error, should be working now

**Review by Codubee — APPROVED** [review]
LGTM!

**Pull request merged**